app/src/main/java/de/danoeh/antennapod/adapter/CoverLoader.java

```diff
@@ -146,17 +148,18 @@ static void setPlaceholderVisibility(TextView placeholder, boolean textAndImageC
             boolean showTitle = UserPreferences.shouldShowSubscriptionTitle();
             if (placeholder != null) {
                 if (textAndImageCombined || showTitle) {
-                    int bgColor = placeholder.getContext().getResources().getColor(R.color.feed_text_bg);
+                    final Resources resources = placeholder.getContext().getResources();
+                    int bgColor = ResourcesCompat.getColor(resources, R.color.feed_text_bg, null);
```

Comment:
`ContextCompat` would be a lot less verbose and easier to understand


```suggestion
                    int bgColor = ContextCompat.getColor(placeholder.getContext(), R.color.feed_text_bg);
```

Reply:
Sure.